modules/claims_api/app/controllers/claims_api/base_form_controller.rb

```diff
@@ -52,5 +52,12 @@ def form_attributes
 
       payload_attributes
     end
+
+    def validate_526_payload
+      valid_service = EVSS::DisabilityCompensationForm::ServiceAllClaim.new(auth_headers)
```

Comment:
Maybe name this just service? `valid_service` is kind of a misnomer, or go full bore and call it `validate_service`